package.json

```diff
@@ -17,11 +17,12 @@
     "node": ">=6"
```

Comment:
I would suggest to update the minimum node 6 engine to 12 for mocha minimum supported version on version 9.